x/dualstaking/keeper/migrations.go

```diff
@@ -74,14 +79,17 @@ func (m Migrator) HandleProviderDelegators(ctx sdk.Context) error {
 	delegationsInds := m.keeper.delegationFS.GetAllEntryIndices(ctx)
 	nextEpoch := m.keeper.epochstorageKeeper.GetCurrentNextEpoch(ctx)
 
-	// move all funds from unbonded pool to bonded pool
-	notBondedPoolAddr := m.keeper.accountKeeper.GetModuleAddress(dualstakingtypes.NotBondedPoolName)
-	notBondedPoolAmount := m.keeper.bankKeeper.GetBalance(ctx, notBondedPoolAddr, epochstoragetypes.TokenDenom)
-	if !notBondedPoolAmount.IsZero() {
-		err := m.keeper.bankKeeper.SendCoinsFromModuleToModule(ctx, dualstakingtypes.NotBondedPoolName, dualstakingtypes.BondedPoolName, sdk.Coins{notBondedPoolAmount})
-		if err != nil {
-			return err
-		}
+	// burn all coins from the pools
+	moduleBalance := m.keeper.bankKeeper.GetBalance(ctx, m.keeper.accountKeeper.GetModuleAddress(dualstakingtypes.NotBondedPoolName), epochstoragetypes.TokenDenom)
+	err := m.keeper.bankKeeper.BurnCoins(ctx, dualstakingtypes.NotBondedPoolName, sdk.NewCoins(moduleBalance))
+	if err != nil {
+		return err
+	}
+
+	moduleBalance = m.keeper.bankKeeper.GetBalance(ctx, m.keeper.accountKeeper.GetModuleAddress(dualstakingtypes.NotBondedPoolName), epochstoragetypes.TokenDenom)
```

Comment:
why are we burning twice the same NotBondedPoolName looks like a mistake

Reply:
done